command/oauth/cmd.go

```diff
@@ -558,13 +558,20 @@ func (o *oauth) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	code := q.Get("code")
+	code, state := q.Get("code"), q.Get("state")
+
+	if code == "" || state == "" {
+		fmt.Printf("Invalid request received from: %v%v\n\n", req.RemoteAddr, req.RequestURI)
+		if req.RequestURI == "/robots.txt" {
+			fmt.Printf("** You may have an app or browser plugin that needs to be turned off **\n\n")
+		}
+	}
```

Comment:
Just logging the issue won't avoid the server shutdown. Here we should return a 403 or 404 in those cases, but without using the badRequest method, as it will send a signal to stop the server. 
We should also use os.Stderr to log those errors.